_locales/_untranslated_en.json

```diff
@@ -91,5 +91,26 @@
   },
   "accounts_list_item_menu_label": {
     "message": "Actions with Profile"
+  },
+  "call_canceled": {
+    "message": "Call canceled"
+  },
+  "call_declined": {
+    "message": "Call declined"
+  },
+  "call_incoming": {
+    "message": "Incoming call"
+  },
+  "call_missed": {
+    "message": "Missed call"
+  },
+  "call_outgoing": {
+    "message": "Outgoing call"
+  },
+  "channel_left_by_you": {
+    "message": "You left the Channel"
+  },
+  "qrshow_join_channel_hint": {
+    "message": "Scan to join Channel."
```

Comment:
It is pointless to add these to `untranslated.json` because the core apparently provides the default English strings.

https://github.com/chatmail/core/blob/9ec0332483a56b97d0a5a4b00e6ae642cdc670c8/src/stock_str.rs#L428-L444

Or maybe it would work for "future-proofing" where we wait for https://github.com/deltachat/deltachat-android/pull/3785 to get merged, which would add these strings, but then it's not that much of future proofing, because the string names (keys) don't match and we'll still have to update the code.